blogs/tekton-oadp-nonadmin/oadp-tekton-container/get_restore_namespace.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#$1 = backup name
+#$2 = allowed projects string
+#
+
+namespace=`oc get backup $1 -o jsonpath='{.spec.includedNamespaces[0]}' -n openshift-adp`
+if [ "$namespace" = "" ]; then
+  echo "Error: backup not found or no namespace associated with backup"
+  exit 1
+fi
+
+if grep -q "$namespace" <<< "$2"; then
+  echo "Found project $namespace in the allowed list of projects for the user"
+else
+  echo "The project $namespace is not allowed for this user"
```

Comment:
If the restore calls for restoring to a different namespace, this may be the wrong check. Do we need to take namespace mappings into account, or are we explicitly excluding namespace mappings from support here? If the latter, do we verify that the restore created won't include namespace mappings?

Reply:
@weshayutin if it's supported (even as advanced) then the above code is wrong, since it's checking permissions based on source namespace, not target namespace. Or would this check not run in the advanced case you're talking about?